docs/api-reference/next/router.md

```diff
@@ -154,6 +154,55 @@ Router.replace('/home')
 
 The API for `Router.replace` is exactly the same as that used for [`Router.push`](#router.push).
 
+### Router.prefetch
+
+Prefetch pages for faster client-side transitions. This method is only useful for navigations without [`next/link`](/docs/api-reference/next/link.md), as the later takes care of prefetching pages for you.
+
+> this is a production only feature, Next.js doesn't prefetch pages on development.
+
+```jsx
+import Router from 'next/router'
+
+Router.prefetch(url, as)
+```
+
+- `url` - The path to a `page`. that is, the name of a page inside the `pages` directory
```

Comment:
If a page is nested, e.g. `pages/a/b/c/d.js` then "name" is not necessarily the right word. So maybe:

```suggestion
- `url` - The path to a `page` inside the `pages` directory
```